README.md

```diff
@@ -1,742 +1,83 @@
-the Compact Thrift API port. You can configure the Jaeger exporter by following the directions below:
-
-1. [Get Jaeger][jaeger-get-started].
-2. Configure the `JaegerExporter`
-    - `ServiceName`: The name of your application or service.
-    - `AgentHost`: Usually `localhost` since an agent should usually be running on the same machine as your application or service.
-    - `AgentPort`: The compact thrift protocol port of the Jaeger Agent (default `6831`)
-    - `MaxPacketSize`: The maximum size of each UDP packet that gets sent to the agent. (default `65000`)
-3. See the [sample][jaeger-sample] for an example of how to use the exporter.
-
-```csharp
-using (var tracerFactory = TracerFactory.Create(
-    builder => builder.UseJaeger(o =>
-    {
-        o.ServiceName = "jaeger-test";
-        o.AgentHost = "<jaeger server>";
-    })))
-{
-    var tracer = tracerFactory.GetTracer("jaeger-test");
-    using (tracer.StartActiveSpan("incoming request", out var span))
-    {
-        span.SetAttribute("custom-attribute", 55);
-        await Task.Delay(1000);
-    }
-}
+TBD
 ```
 
-### Using Zipkin exporter
+## Documentation
 
-Configure Zipkin exporter to see traces in Zipkin UI.
+The online documentation is available at TBD.
 
-1. Get Zipkin using [getting started guide][zipkin-get-started].
-2. Configure `ZipkinTraceExporter` as below:
-3. See [sample][zipkin-sample] for example use.
-
-```csharp
-using (var tracerFactory = TracerFactory.Create(builder => builder
-    .UseZipkin(o =>
-    {
-        o.ServiceName = "test-zipkin";
-        o.Endpoint = new Uri(zipkinUri);
-    })))
-{
-    var tracer = tracerFactory.GetTracer("zipkin-test");
-
-    // Create a scoped span. It will end automatically when using statement ends
-    using (tracer.WithSpan(tracer.StartSpan("Main")))
-    {
-        Console.WriteLine("About to do a busy work");
-        for (var i = 0; i < 10; i++)
-        {
-            DoWork(i, tracer);
-        }
-    }
-}
-```
-
-### Using Prometheus exporter
-
-Configure Prometheus exporter to have stats collected by Prometheus.
-
-1. Get Prometheus using [getting started guide][prometheus-get-started].
-2. Start `PrometheusExporter` as below.
-3. See [sample][prometheus-sample] for example use.
-
-```csharp
-var exporter = new PrometheusExporter(
-    new PrometheusExporterOptions()
-    {
-        Url = "http://+:9184/metrics/"
-    },
-    Stats.ViewManager);
-
-exporter.Start();
-
-try
-{
-    // record metrics
-    statsRecorder.NewMeasureMap().Put(VideoSize, values[0] * MiB).Record();
-}
-finally
-{
-    exporter.Stop();
-}
-```
-### Using the NewRelic exporter
+## Compatible Exporters
 
-The New Relic OpenTelemetry Trace Exporter is a OpenTelemetry Provider that sends data from .NET applications to New Relic.
-It uses the NewRelic SDK to send Traces to the New Relic backend
+See the [OpenTelemetry registry](https://opentelemetry.io/registry/?s=net) for a list of exporters available.

```

Comment:
We will need a follow up since https://opentelemetry.io/registry/?s=net is not up to date.

Reply:
Definitely! This can be covered when we clean up the exporters, tracked by #645 #646 #647.